scripts/dev/generate_epJSON_schema/idd_parser.py

```diff
@@ -725,6 +726,18 @@ def eat_whitespace(data):
             break
 
 
+def parse_group(data):
```

Comment:
I rolled a slightly modified parser worker function that eats until the end of the line.  This could certainly eat the `\group ` part if that's the preferred approach.  If there is another worker function that could also do this, I'm all for it.